sass/ohos/_wikidata-overlays.scss

```diff
@@ -0,0 +1,84 @@
+@use "@nationalarchives/frontend/nationalarchives/tools/typography";
+
+.wikidata {
+    --spacer-size: 1em;
```

Comment:
Is there a benefit to this being a CSS var?

If we aren't planning to change it in the browser, should this be a SCSS variable instead?